src/network/http/http_client.cpp

```diff
@@ -350,10 +376,45 @@ class http_client_impl {
       return result;
    }
 
+   /*
+      Unix URLs work a little special here. They'll originally be in the format of
+      unix:///home/spoonincode/eosio-wallet/keosd.sock/v1/wallet/sign_digest
+      for example. When the fc::url is given to http_client in post_sync(), this will
+      have proto=unix and host=/home/spoonincode/eosio-wallet/keosd.sock/v1/wallet/sign_digest
```

Comment:
Generic username path?

Reply:
I don't follow? obviously (or, at least I though obviously) that comment is meant to be an example

Reply:
In the context of Github, it obviously is an example.  In bare source code, I didn't think it was completely obvious that 'spoonincode' is someone's username.

`host=/home/username/eosio-wallet/keosd.sock/v1/wallet/sign_digest` is more generic.  Likewise for two lines above.